windows/src/main/csharp/ch/cyberduck/ui/controller/PreferencesController.cs

```diff
@@ -1035,6 +1032,8 @@ private void Init()
             View.Retries = PreferencesFactory.get().getInteger("connection.retry");
             View.RetryDelay = PreferencesFactory.get().getInteger("connection.retry.delay");
             View.ConnectionTimeout = connectionTimeoutPreferences.getTimeout();
+            View.ConnectionTimeoutMin = PreferencesFactory.get().getInteger("connection.timeout.min.seconds");
+            View.ConnectionTimeoutMax = PreferencesFactory.get().getInteger("connection.timeout.max.seconds");
```

Comment:
When getTimeout is 120, this will throw Argument out of range, as by-default Min is 0 and Max is 100.
Assign min and max range before setting value.